src/components/TableCard/TableCard.story.jsx

```diff
@@ -617,6 +617,101 @@ storiesOf('Watson IoT/TableCard', module)
       </div>
     );
   })
+  .add(
```

Comment:
Do we need 2 different stories for this? I think if we have 1 whole story using all of the different variables, that will suffice

Reply:
We do need two if we want to show both, unfortunately, because a table with row specific vars needs to omit the cardVariables prop. I agree that there are a ton of stories.

Reply:
Ah I forgot about that. Nevermind then!